R/task-eval.R

```diff
@@ -66,6 +66,11 @@ task_eval <- function(id, envir = .GlobalEnv, verbose = FALSE, root = NULL) {
       data$id <- id
     }
     envvars_apply(data$envvars, top)
+
+    default_cores <- hipercow_parallel_get_cores()
+    if (is.na(default_cores)) default_cores <- NULL
+    hipercow_parallel_set_cores(default_cores, rlang::current_env())
```

Comment:
```suggestion
    cores <- hipercow_parallel_get_cores()
    if (!is.na(cores)) {
      hipercow_parallel_set_cores(cores, rlang::current_env())
    }
```

* I don't think this is really *default* cores, so much as cores
* the logic currently in set_cores is better here I think, and avoids churn while passing around different types of missingness

Reply:
OK - I think that's better now?